app/views/pages/cookies.html.erb

```diff
@@ -1,78 +1,181 @@
+        <h3 class="govuk-heading-s">Cookies that measure website use</h3>
+        <p>We use Google Analytics to measure how you use the website so we can improve it based on user needs. We do not allow Google to use or share the data about how you use this site.</p>
+        <p>Google Analytics sets cookies that store anonymised information about:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>how you got to the site</li>
+          <li>the pages you visit on the UK Global Online Tariff, and how long you spend on each page</li>
+          <li>what you click on while you're visiting the site</li>
         </ul>
+        <p>Google Analytics sets the following cookies:</p>
+        <table class="govuk-table govuk-table--m">
+          <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Name</th>
+            <th scope="col" class="govuk-table__header">Purpose</th>
+            <th scope="col" class="govuk-table__header">Expires</th>
+          </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_ga</td>
+            <td class="govuk-table__cell">These help us count how many people visit the service by tracking if you’ve visited before</td>
+            <td class="govuk-table__cell">2 years</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_gid</td>
+            <td class="govuk-table__cell">This randomly generated number is used to determine unique visitors to our site</td>
+            <td class="govuk-table__cell">24 hours</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_gat_govuk_shared</td>
+            <td class="govuk-table__cell">This randomly generated number is used to reduce the number of requests made</td>
+            <td class="govuk-table__cell">2 years</td>
+          </tr>
+          </tbody>
+        </table>
+        <br>
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset" aria-describedby="cookie-consent-usage-hint">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xs">
+              Do you consent to cookies that measure website use?
+            </legend>
+            <div id="cookie-consent-usage-hint" class="govuk-hint">
+            </div>
+            <div class="govuk-radios govuk-radios--stacked">
+              <div class="govuk-radios__item">
+                <%= form.radio_button :cookie_consent_usage, true, class: "govuk-radios__input",
+                                      checked: cookies[:cookies_policy] && JSON.parse(cookies[:cookies_policy])['usage'] == true %>
+                <%= form.label(:cookie_consent_usage_1, "use cookies that measure my website use",
```

Comment:
Feels like `JSON.parse(cookies[:cookies_policy])['usage'] == true` should stay on a helper method maybe? Say inside  cookies_helper.rb.